pages/donghuas/_donghua.vue

```diff
@@ -42,7 +42,7 @@
         <a-tab-pane key="1" tab="Media"></a-tab-pane>
         <a-tab-pane key="2" tab="Related Works"></a-tab-pane>
         <a-tab-pane key="3" tab="Staff">
-          <h4 v-for="staff in staffs" :key="staff">{{ staff.name }}</h4>
+          <a-table :columns="columns" :data-source="data" bordered></a-table>
```

Comment:
in case other tables get added later, I think it would be better to give the variables more specific names eg. `staffTableColumns` and `staffTableData`

Reply:
I see

Reply:
Is it correct to use **Person Name** as column head?

Reply:
Oh yeah for the column header, I think just having it labelled `Name` would be better